test/unit/util/test_cuda_vector_matrix.cpp

```diff
@@ -446,3 +447,26 @@ TEST(CudaVectorMatrix, AssignmentFromVector)
   EXPECT_EQ(matrix[2][2], 65.7);
   EXPECT_EQ(matrix[3][0], 0.0);
 }
+
+TEST(CudaVectorMatrix, Axpy)
+{
+  const double alpha = 2.0;
+
+  // Generate a 20 x 10 matrix with all elements set to 10.0
+  auto gpu_x = micm::CudaVectorMatrix<double, 10>(20,10,10.0);
+  auto gpu_y = micm::CudaVectorMatrix<double, 10>(20,10,20.0);
+  gpu_x[0][1]  = 20.0;
+  gpu_x[1][1]  = 30.0;
+
+  gpu_x.CopyToDevice();
+  gpu_y.CopyToDevice();
+  gpu_y.Axpy(alpha, gpu_x, 1, 1);
+  gpu_y.CopyToHost();
+
+  EXPECT_EQ(gpu_y[0][0], 40.0);
+  EXPECT_EQ(gpu_y[0][1], 60.0);
+  EXPECT_EQ(gpu_y[1][1], 80.0);
+
+  double sum = std::accumulate(gpu_y.AsVector().begin(), gpu_y.AsVector().end(), 0);
+  EXPECT_EQ(sum, (10*2.0+20.0)*198 + 20.0*2.0+20.0 + 30.0*2.0+20.0);
+}
```

Comment:
```suggestion
}

```
Missing newline at end of file.

Reply:
How could you catch this!

Reply:
😃 GitHub has an annoying red X on the end of files in the Files Changed view.  I've caught myself so many times post commit and having to push up a bunch of newline fixes.

Thanks for fixing!